tests/compat/test_gecko40.py

```diff
@@ -46,13 +46,6 @@ def test_extIApplication(self):
         self.assert_silent()
         self.assert_compat_warning(type_='warning')
 
-    def test_Application(self):
```

Comment:
The validation currently does not support multiple checks for the same instance call. But as compat tests for 40 ran a long time ago and won't run again, we should be ok.